trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilityType.ts

```diff
@@ -0,0 +1,54 @@
+import { Facility, FacilityType, FACILITY_TYPE, MAPPED_FACILITY_TYPE } from '@ukef/dtfs2-common';
+
+const { BOND, LOAN, CASH, CONTINGENT } = FACILITY_TYPE;
+
+/**
+ * Maps type to a different facility type used in TFM-UI and TFM-API.
```

Comment:
```suggestion
 * Maps a facility type to a differently worded facility type,
 * to ultimately be rendered in the UI, as per design.
```

Or similar - just so it's clear _why_ we're doing this